tests/templates/get_ansible_managed.j2

```diff
@@ -0,0 +1 @@
+{{ ansible_managed | comment }}
```

Comment:
can this use the template from the role?

Reply:
Instead, updated to use a hardcoded string in the tests following the suggestion from @tyll.